linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_max_life_existing/bash/shared.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/bash
 
-# platform = multi_platform_sle
+# platform = Red Hat Virtualization 4,multi_platform_fedora,multi_platform_ol,multi_platform_rhel,multi_platform_sle
 # reboot = false
 # strategy = restrict
 # complexity = low
 # disruption = low
 
-SECURE_MAX_PASS_AGE=60
+{{{ bash_instantiate_variables("var_accounts_maximum_age_login_defs") }}}
 
-usrs_max_pass_age=( $(awk -F: '$5 > SECURE_MAX_PASS_AGE || $5 == "" {print $1}' /etc/shadow) )
+usrs_max_pass_age=( $(awk -v var="$var_accounts_maximum_age_login_defs" -F: '($2 != "!!" || $2 != "*" ) && ($5 > var || $5 == "") {print $1}' /etc/shadow) )
```

Comment:
This should actually be:
```suggestion
usrs_max_pass_age=( $(awk -v var="$var_accounts_maximum_age_login_defs" -F: '($2 != "!!" && $2 != "*" ) && ($5 > var || $5 == "") {print $1}' /etc/shadow) )
```
But we can wait on whether it is preferred to set min/max age on all accounts, regardless if a password is set or not.